examples/svelte/src/App.svelte

```diff
@@ -0,0 +1,66 @@
+<script>
+    import { router } from './router.js';
+    import { uuid } from './utils.js';
```

Comment:
we can use the platform here

```suggestion
```

Reply:
ah - now it makes sense, after reading all comments - please see my note below